### DISCUSSION TITLE
Error processing send-email job. Reason: certificate has expired

### DISCUSSION THREAD

**Discussion by antitiron** [docmost/docmost#66] [Help / Questions]
Hello!

When sending emails I get an error "Error processing send-email job. Reason: certificate has expired".

The certificate for smtp is exactly extended until October 3, 2024.

![image](https://github.com/docmost/docmost/assets/7191601/46af7c3c-25b3-40dd-a4a6-09dbd533fc8a)

The settings are as follows:

```
MAIL_DRIVER: smtp
SMTP_HOST: mail.domain.com
SMTP_PORT: 587
SMTP_USERNAME: username@domain.com
SMTP_PASSWORD: password
MAIL_FROM_ADDRESS: docmost@domain.com
MAIL_FROM_NAME: Docmost
```

**Comment by Philipinho** [comment]
Which SMTP email provider are you using?

**Comment by antitiron** [comment]
[Maddy Mail Server](https://maddy.email/) on my host
